### DISCUSSION TITLE
Your GitHub account does not have verified academic email address

### DISCUSSION THREAD

**Discussion by ElkeDeZitter** [RosettaCommons/rosetta#220] [ROSIE] [answered]
Hi,
My institutional e-mail address is(@ibs.fr) is not recognised as academic.
Could you please add it?
Thanks in advance, Elke

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know! This should be fixed now so please give it another try. Please feel free to re-open this issue if you still having trouble login-in. Thanks,

**Reply by ElkeDeZitter** [reply]
Thanks!
